src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -473,6 +487,106 @@ public Void call() throws Exception {
             }
             return null;
         }
+
+        /**
+         * Logs supplied StringBuilder to supplied Logger and Writer.
```

Comment:
@dean-e-clark this sounds a bit strange, we are not supplying any `Logger` here, just reader and writer